starfish/starfish.py

```diff
@@ -141,15 +140,21 @@ def detect_spots(in_json, results_dir, aux_img, min_sigma, max_sigma, num_sigma,
     s = Stack()
     s.read(in_json)
 
-    gsp = GaussianSpotDetector(s.squeeze(), s.aux_dict[aux_img])
+    # create 'encoder table' standard (tidy) file format.
+    gsp = GaussianSpotDetector(s)
+    spots_df_tidy = gsp.detect(min_sigma=min_sigma,
```

Comment:
not a requirement, but if you do:

```
spots_df_tidy = gsp.detect(
        min_sigma=min_sigma,
        ...,
)
```

you avoid having to reformat all the lines in the future should you choose to rename a function name or variable, such that your open paren moves.